docs/running-a-node/docker-management.md

```diff
@@ -165,6 +165,20 @@ Usage:
 ```
 ---
 
+### Docker USER Support
+
+As of v20.0, the docker containers support the [--user=](https://docs.docker.com/engine/reference/run/#user) and [-w=](https://docs.docker.com/engine/reference/run/#workdir) flags.
+
+To maintain existing compatibility the Docker containers are being built with `USER ROOT` and `WORK_DIR /root`
+
+The problem with this is that the container ends up writing files to your mounted path as root. Best practices would dictate that since there is no need for privilage escalation we can instead create a user and run under that context instead.
```

Comment:
Spelling and wording tweak: "Best practices would dictate that since there is no need for privilege escalation we can create a user and run under that context instead."